relayer/src/setup/index.ts

```diff
@@ -1,36 +1,37 @@
-import {
-  createTestAccounts,
-  initLookUpTable,
-  useWallet,
-} from "@lightprotocol/zk.js";
-import { getAnchorProvider, getKeyPairFromEnv } from "../utils/provider";
-import { PublicKey } from "@solana/web3.js";
-import { readFileSync, writeFileSync } from "fs";
-import { RPC_URL } from "../config";
-
-export async function relayerSetup() {
-  const anchorProvider = await getAnchorProvider();
-
-  await createTestAccounts(anchorProvider.connection);
+import { initLookUpTable, useWallet } from "@lightprotocol/zk.js";
+import { getKeyPairFromEnv } from "../utils/provider";
+import { AddressLookupTableAccount, PublicKey } from "@solana/web3.js";
+import { getLookUpTableVar, setLookUpTableVar, RPC_URL } from "../config";
+import { AnchorProvider } from "@coral-xyz/anchor";
 
+export async function setupRelayerLookUpTable(anchorProvider: AnchorProvider) {
   let lookUpTable;
-  const path = "lookUpTable.txt";
+
   try {
-    let lookUpTableRead = new PublicKey(readFileSync(path, "utf8"));
+    let lookUpTableRead = new PublicKey(getLookUpTableVar()!);
+    console.log("lookUpTableRead::", lookUpTableRead);
     let lookUpTableInfoInit = await anchorProvider.connection.getAccountInfo(
       lookUpTableRead,
     );
     if (lookUpTableInfoInit) {
       lookUpTable = lookUpTableRead;
     }
+    AddressLookupTableAccount.deserialize(lookUpTableInfoInit!.data);
   } catch (e) {
-    console.log(".txt not found", e);
+    console.log(".look_up_table env not found or not properly initialized", e);
   }
   if (!lookUpTable) {
     console.log("initing lookuptable...");
     let wallet = useWallet(getKeyPairFromEnv("KEY_PAIR"), RPC_URL);
 
     lookUpTable = await initLookUpTable(wallet, anchorProvider);
-    writeFileSync(path, lookUpTable.toString(), "utf8");
+    console.log("new relayer lookUpTable created: ", lookUpTable.toString());
+    setLookUpTableVar(lookUpTable.toString());
+    console.log("putting var into cache:", getLookUpTableVar());
+
+    console.log(
+      ">> please set LOOK_UP_TABLE env var to:",
+      lookUpTable.toString(),
+    );
```

Comment:
let's just keep one lookupt table print, there is three more in line 30, 28, and 12.